src/main/webapp/WEB-INF/view/activity.jsp

```diff
@@ -54,18 +60,24 @@ MessageStore messageStore = (MessageStore) request.getAttribute("messageStore");
 <div id="feed">
   <ul>
     <%
-      /* Get the titles of the conversations. */
+       /* Used to format the time since the getCreationTime() method returns an instance, not a string. */
+       DateTimeFormatter formatter =DateTimeFormatter.ofLocalizedDateTime( FormatStyle.SHORT ).withLocale( Locale.US ).withZone( ZoneId.systemDefault() );
```

Comment:
nit:
Try not to have very long line such as this one. Reviewer has to scroll to the right to see the rest of the line.
Seems the width here is about 120 chars (before scrolling), so try to keep line width below that.

You can use multiple lines such as:
DateTimeFormatter formatter =
      DateTimeFormatter.ofLocalizedDateTime(FormatStyle.SHORT)
          .withLocale(Locale.US)
          .withZone(ZoneId.systemDefault());